zerver/tests/test_events.py

```diff
@@ -2028,6 +2028,9 @@ def test_do_delete_message(self) -> None:
             ('type', equals('delete_message')),
             ('message_id', check_int),
             ('sender', check_string),
+            ('is_stream_message', check_bool),
```

Comment:
You can use things like `equals("stream")`  here to check that it produced the right value; would be good when this is split into two tests.